setup.ts

```diff
@@ -193,14 +206,18 @@ async function mongoDB(): Promise<void> {
   let DB_URL = process.env.MONGO_DB_URL;
 
   try {
-    let isConnected = false,
-      url = "";
+    let isConnected = await checkLocalMongoDB(),
+      url = isConnected ? "mongodb://localhost:27017" : "";
```

Comment:
Why have you hard coded localhost? Is this only implemented when there is no `MONGO_DB_URL` configured?

Reply:
I have hardcoded localhost only when the script detects a local instance running, used ternary operator to check that

Reply:
1. Does the script then use the detected URL as the default?
2. So therefore if the user hits enter  then `mongodb://localhost:27017/talawa-api` will be used?

Reply:
If there is a non localhost URL configured, will that be used or will the localhost detected instance be the value presented?

Reply:
Yes now, it will use `mongodb://localhost:27017/talawa-api` as the default url if local instance is found running on their machine. Otherwise it will either use the url sepicfied in .env or promt them for url